### ISSUE TITLE
Pairing error.

### ISSUE THREAD

**Issue by shogochiai** [Samourai-Wallet/whirlpool-gui#6]
Just for reporting.

`Authentication failed: Unable to initialize Bip84 wallet` error is shown.

![Screen Shot 2019-07-03 at 18 57 17](https://user-images.githubusercontent.com/4891004/60582876-9e824880-9dc4-11e9-9441-a4136ef228db.png)

I'm using `Segwit compatibility (P2SH)` wallet because it was by default on `Samourai, 0.99.74 (Android)`.

**Comment by SamouraiDev** [comment]
Update to the latest Play version and try again. Thank you.

**Comment by zeroleak** [comment]
This may also be a problem with TOR, try to disable it in configuration and retry.
Can you check "CLI errors" log available in "System" (whirlpool-cli.error.log)?

**Comment by Head** [comment]
same here, without Tor it works.

(TOR all caps is wrong, see https://2019.www.torproject.org/docs/faq.html.en#WhyCalledTor)

```
[ERROR]2019-07-07 07:22:25.570 ERROR 3620 --- [           main] com.samourai.http.client.JavaHttpClient  : getJson failed: https://api.samouraiwallet.com/v2/fees

com.samourai.whirlpool.client.exception.NotifiableException: Tor connect failed
	at com.samourai.tor.client.TorOnionProxyInstance.waitReady(TorOnionProxyInstance.java:74) ~[classes!/:na]
	at com.samourai.tor.client.TorOnionProxyInstance.getTorProxy(TorOnionProxyInstance.java:155) ~[classes!/:na]
	at com.samourai.whirlpool.cli.utils.CliUtils.computeHttpClient(CliUtils.java:155) ~[classes!/:na]
	at com.samourai.http.client.JavaHttpClient.computeHttpClient(JavaHttpClient.java:122) ~[classes!/:na]
	at com.samourai.http.client.JavaHttpClient.getJson(JavaHttpClient.java:39) ~[classes!/:na]
	at com.samourai.wallet.api.backend.BackendApi.fetchFees(BackendApi.java:92) [extlibj-0.0.4.jar!/:na]
	at com.samourai.api.client.SamouraiApi.testConnectivity(SamouraiApi.java:30) [whirlpool-client-0.21.5.jar!/:na]
	at com.samourai.whirlpool.cli.services.CliPushTxService.testConnectivity(CliPushTxService.java:57) [classes!/:na]
	at com.samourai.whirlpool.cli.Application.runCli(Application.java:172) [classes!/:na]
	at com.samourai.whirlpool.cli.Application.run(Application.java:109) [classes!/:na]
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:804) [spring-boot-2.1.4.RELEASE.jar!/:2.1.4.RELEASE]
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:794) [spring-boot-2.1.4.RELEASE.jar!/:2.1.4.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:324) [spring-boot-2.1.4.RELEASE.jar!/:2.1.4.RELEASE]
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:139) [spring-boot-2.1.4.RELEASE.jar!/:2.1.4.RELEASE]
	at com.samourai.whirlpool.cli.Application.main(Application.java:86) [classes!/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_211]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[na:1.8.0_211]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[na:1.8.0_211]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[na:1.8.0_211]
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48) [whirlpool-client-cli-0.7.0-run.jar:na]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87) [whirlpool-client-cli-0.7.0-run.jar:na]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50) [whirlpool-client-cli-0.7.0-run.jar:na]
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51) [whirlpool-client-cli-0.7.0-run.jar:na]```

**Comment by zeroleak** [comment]
New update available (CLI 0.7.1) which may fix your issue. 
(if using GUI, restart GUI to update automatically)
